manifests/quick-start/sso/overlays/workflow-controller-configmap.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 data:
   sso: |
     issuer: http://dex:5556/dex
-    issuerAlias: http://mydex:5556/dex
+    issuerAlias: http://dex:5556/dex
```

Comment:
SSO doesn't work with `http://mydex:5556/dex` in local environment
Hence this change